src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -1150,10 +1178,25 @@ protected async Task<TResult> InvokeCoreAsync<TResult>(long? id, string targetNa
                                 {
                                     tcs.TrySetException(CreateExceptionFromRpcError(error, targetName));
                                 }
+
+                                if (JsonRpcEventSource.Instance.IsEnabled())
+                                {
+                                    responseDetails = JsonRpcEventSource.GetErrorDetails(error);
+                                }
                             }
                             else if (response is JsonRpcResult result)
                             {
                                 tcs.TrySetResult(result.GetResult<TResult>());
+
+                                if (JsonRpcEventSource.Instance.IsEnabled())
+                                {
+                                    responseDetails = $"Success!, Result.Id = \"{result.Id}\"";
+                                }
+                            }
+
+                            if (JsonRpcEventSource.Instance.IsEnabled())
+                            {
+                                JsonRpcEventSource.Instance.SendRequestStop($"ResponseDetails: {responseDetails}; RequestDetails: Id: \"{request.Id}\", Method: \"{request.Method}\", Arguments: \"{JsonRpcEventSource.GetArgumentsString(request.Arguments)}\"");
```

Comment:
For some reason, Duration_Msec is not getting captured for most of the SendRequestStop events. For such events, no ActivityId shows up in PerfView. 
In case of WinForms designer, we spawn a .NetCore process from devenv. Not sure if framework mismatch is the issue.

Reply:
That's probably because of missing attribute properties. See https://github.com/microsoft/vs-streamjsonrpc/pull/322#discussion_r314906719

Reply:
Tried but it didn't help.